# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/all.js
# Commit: 0310d86a37b5
# Full Hash: 0310d86a37b5a00d1561a186832604c35cdd40d9
# Author: Tom Ritter <tom@mozilla.com>
# Date: 2021-08-24 21:59:48
# Regressor Bug: 1723204
# File Overlap Count: 1
# Description:
#   Bug 1723204: Wire up a crash for Javascript Load Telemetry r=ckerschb,Gijs
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D121417
# ==============================================================================

diff -r 6b6c21180ed9 -r 0310d86a37b5 modules/libpref/init/all.js
--- a/modules/libpref/init/all.js	Tue Aug 24 14:57:43 2021 +0000
+++ b/modules/libpref/init/all.js	Tue Aug 24 14:57:44 2021 +0000
@@ -213,6 +213,9 @@
 pref("security.osreauthenticator.password_last_changed_lo", 0);
 pref("security.osreauthenticator.password_last_changed_hi", 0);
 
+pref("security.crash_tracking.js_load_1.prevCrashes", 0);
+pref("security.crash_tracking.js_load_1.maxCrashes", 1);
+
 pref("general.useragent.compatMode.firefox", false);
 
 pref("general.config.obscure_value", 13); // for MCD .cfg files
